### ISSUE TITLE
Dynamically change the height of a field

### ISSUE THREAD

**Issue by Slijkhuis** [nicklockwood/FXForms#319]
When I subclass `FXFormBaseCell`, can I change the height of the table row in the `update` method? If not, where am I able to change it?

**Comment by nicklockwood** [comment]
There's a +heightForField: method you can override that is called whenever the row is updated.

**Comment by Slijkhuis** [comment]
Ow, missed that! Thanks!

I need to display an image in the row. I thought it was hidden because the height of the row wasn't sufficient. However, I now see that (since I'm able to set the height now) it's not displaying at all. What would be the recommended way to display an image in the table row?

Currently, I'm adding the image by adding a `UIImageView` as a subview to `contentView` (placing it at the x/y position of `detailTextLabel`). But that does not seem to be working.

**Comment by nicklockwood** [comment]
I'm not sure why that wouldn't work, but take a look at how FXFormImagePickerCell works, and see if that's doing anything differently.

You might also try using Xcode's view logging features to inspect your cell and see if the imageView is being wrongly positioned or sized, or if the image property has been set correctly.

**Comment by tosbaha** [comment]
@Slijkhuis You can also add image like this
```swift
    func nameField() -> NSDictionary {
        let image:UIImage = UIImage(named: "user")!
        return ["imageView.image": image]
    }
```
However, be aware that padding of cells are not calculated correctly if there is an image in standard cell. Especially in TextView cells, image collapses with content. You can change the origin of textView frame like this 
```swift
textViewFrame.origin.x = 2*FXFormFieldPaddingLeft + self.imageView.frame.size.width;
```
Although it is ugly hack, it worked for my purposes. I hope @nicklockwood adds support for imageview support for other basic cells as well.

**Comment by Slijkhuis** [comment]
Nice, thanks for the tip. However, I'm not having trouble with displaying an image from the start. The problem only occurs when I want to change the image based on user input.

I haven't found the exact problem yet, but I'll it post here as soon as I find out where it goes wrong. I suspect it has something to do with the tableview not redrawing.

**Comment by Slijkhuis** [comment]
Never mind, it suddenly started working... I really don't know why.

First, I set the image like this:

1. `- (void)didSelectWithTableView:(UITableView *)tableView controller:(UIViewController *)controller` is called.
2. A view controller is pushed to front
3. After user interaction, another view controller is pushed to the front
4. After async contact with the server, the view controllers are popped and the image is set and the `self.field.value` is set.

Now it's like this:
1/2/3. same
4. Same, but only `self.field.value` is set and in the `[self update]` method I set the image *based on `self.field.value`*

I'm not sure what the difference is.. But it's working now :).
